common/src/View/MapViewBase.cpp

```diff
@@ -1037,7 +1037,7 @@ namespace TrenchBroom {
 
             addMainMenuAction(IO::Path("Menu/Edit/Create Linked Duplicate"));
             addMainMenuAction(IO::Path("Menu/Edit/Select Linked Groups"));
-            addMainMenuAction(IO::Path("Menu/Edit/Unlink Groups"));
+            addMainMenuAction(IO::Path("Menu/Edit/Separate Linked Groups"));
```

Comment:
Maybe `addMainMenuAction` should be made to `assert(false)` if the action isn't found?

Reply:
Yeah, good point. Done!